Automattic/vip-go-mu-plugins
# Update/jetpack9.0.2

## Description

Bumps 9.0 folder to 9.0.2 content. Also bumps the version in root jetpack.php to 9.0.2 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply branch
1. See the Jetpack is on 9.0.2
1. Check some basic operation and that wp-admin and frontend both loads fine.
